### PR TITLE
Metrics forwarding to logs

### PR DESCRIPTION
**Pull request by PierreBesson**
Enable metrics forwarding to ELK. Linked to #3082

### CODE REVIEW

File: generators/server/templates/src/main/resources/config/_application-prod.yml [hunk 1]
```diff
@@ -181,6 +181,9 @@ jhipster:
             host: localhost
             port: 2003
             prefix: <%= baseName %>
+        logs: # report metrics in the logs
+            enabled: false
+            reportFrequency: 60 # in seconds
```
Comment (deepu105) [inline generators/server/templates/src/main/resources/config/_application-prod.yml]: does doing this every minute have any performance impact? especially in prod when enabled?

Comment (PierreBesson) [inline generators/server/templates/src/main/resources/config/_application-prod.yml]: Actually I don't really know what default frequency should be used. The spark reporter also report logs every minute and does not even provide a way to configure the frequency in the app properties.
Anyway, It should not have too much performance impact as it is only sending a few dozens of logs every minute minute.

Comment (deepu105) [inline generators/server/templates/src/main/resources/config/_application-prod.yml]: Ok was just curious :)

Thanks & Regards,
Deepu

**Comment by pascalgrimaud** [comment]
@PierreBesson : what do you think about adding your `elk.yml` too ?
I thought it was already done to generator-jhipster project but I don't see this file 

It will be easy to test, right ?
- change config false to true
- start elk with docker-compose
- start microservice

**Comment by PierreBesson** [comment]
Currently, the elk.yml is in a [PR on the registry](https://github.com/jhipster/jhipster-registry/pull/15).
I didn't want to clutter every apps with this so I didn't put this in the main generator.
Actually, the new idea is to write a Docker Compose generator to start all the micro-services at once (see #2988). I will put the elk.yml there.

**Comment by pascalgrimaud** [comment]
Ok I see ! thanks Pierre

**Comment by cbornet** [comment]
The elk.yml would be needed for monolith apps I guess

**Pull request merged**